components/Countdown.js

```diff
@@ -0,0 +1,39 @@
+var React = require("react");
+import PropTypes from "prop-types";
+import timediff from "timediff";
+class Countdown extends React.Component {
+  constructor(props) {
+    super(props);
+    this.tick = this.tick.bind(this);
+    this.state = { date: props.initialDate, now: Date.now() };
+    this.TimeDiff = this.timeDiff();
```

Comment:
Can these 2 be renamed into something more destinguishable and readable. Also, please don’t use CamelCase for instance variables.